contentful/export-processor/.gitignore

```diff
@@ -1 +1,4 @@
 contentful*.json
+content-errors*.md
+plan-tech-test-suite*.csv
+output/
```

Comment:
I'd add the user created .env to this too to save someone accidentally committing it later

Reply:
Good shout. Mine's already being ignored for some reason so I didn't realise it wasn't already ignored. I must have it ignored locally somewhere, whoops. 

Sorted!